README.adoc

```diff
@@ -1,4 +1,4 @@
-= Hawkular Middleware Components for ManageIQ
+= Middleware Components for ManageIQ
```

Comment:
This is still wrong, this repo is for *angular* components, not middleware specific. At least, neither toolbars nor gtl list are middleware in any way..